src/site/includes/header.html

```diff
@@ -67,6 +67,10 @@
     <script nonce="**CSP_NONCE**" src="{{ botframework_cdn }}"></script>
   {% endif %}
 
+  {% if path == 'coronavirus-veteran-frequently-asked-questions' %}
```

Comment:
Would it be easier to maintain if we moved this conditional into the `coronavirus-schema-org.json.liquid` file?

Reply:
If you think so I can move it. I don't have any preference about it

Reply:
I also don't have any preference about it, all good!

Reply:
I agree with @kelsonic

Reply:
Okay, moved